cupy/linalg/_eigenvalue.py

```diff
@@ -141,7 +207,53 @@ def eigh(a, UPLO='L'):
         return _syevd(a, UPLO, True)
 
 
-# TODO(okuta): Implement eigvals
+def eig(a):
```

Comment:
How about transpose the input as follows?
```py
a = a.transpose(0, 2, 1).copy(order='C').transpose(0, 2, 1)
```
The output array can also be constructed in the same way.